Sources/SPTDataLoaderServiceSessionSelector.m

```diff
@@ -71,6 +80,18 @@ - (NSURLSession *)waitingSession
     return _waitingSession;
 }
 
+- (NSURLSession *)backgroundSession
+{
+    if (!_backgroundSession) {
```

Comment:
to be consistent with the style in this file, would it be possible to be explicit using `== nil` ?